src/libraries/System.Runtime/tests/TrimmingTests/InvariantGlobalizationTrue.cs

```diff
@@ -13,15 +12,6 @@ class Program
 {
     static int Main(string[] args)
     {
-        // since we are using Invariant GlobalizationMode = true, setting the culture doesn't matter.
-        // The app will always use Invariant mode, so even in the Turkish culture, 'i' ToUpper will be "I"
-        Thread.CurrentThread.CurrentCulture = new CultureInfo("tr-TR");
```

Comment:
Can we find another way to test that Invariant mode is set here? This is basically removing what this test is trying to test.

Maybe a thought here is to split this test into 2?
* One just set Invariant = true, and runs this part of code.
* One sets both Invariant = true && PredefinedCultures = true, and tests that the right things are trimmed from GlobalizationMode.

Reply:
I am not sure what this is providing more what we test in other tests. We have a complete test in the Invariant test project which is testing Invariant mode when PredefinedCultures is set to false. 

When having Invariant mode is on and PredefinedCultures is true, we'll not be able to create any culture tr-TR anyway and we'll throw exception. 

Could you please provide mode details what is we are getting to have tr-TR check here?

Reply:
> I am not sure what this check providing more than what we are doing in other Invariant tests.

No other tests are running against a trimmed app. This is the only test that sets `Invariant=true`, trims the app and then executes it to ensure it still works correctly.

> tr-TR

Using Turkish culture isn't the important thing here. The important thing is testing that Invariant mode is working correctly. We just happened to be using Turkish culture here as a way to test Invariant mode in a trimmed app.

See also the other side of this test, when Invariant=false:

https://github.com/dotnet/runtime/blob/daeee34715ea012d7a42af19137fd73f03b1f151/src/libraries/System.Runtime/tests/TrimmingTests/InvariantGlobalizationFalse.cs#L13-L22

Reply:
@eerhardt I have addressed this in my last commit.